ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -451,17 +467,16 @@ module Vendor_nvidia = struct
     List.filter_map (fun vgpu_type ->
       let is_sriov id =
         try (* IDs of T4 cards *)
-          !Xapi_globs.nvidia_t4_sriov && List.mem (int_of_string id)
+          List.mem (int_of_string id)
             [230; 231; 232; 233; 234; 225; 226;
              227; 228; 229; 222; 252; 223; 224]
         with Not_found | Failure _ -> false
```

Comment:
What can raise Not_found here? Wouldn't List.mem return false?

Reply:
You are right, `Not_found` can't happen. I will remove it.